### ISSUE TITLE
`brew services stop sonarqube` does not stop sonarqube

### ISSUE THREAD

**Issue by VonUniGE** [Homebrew/homebrew-services#395]
**Steps to reproduce:**
````
$ brew services start sonarqube
$ ps aux | grep sonar # Check that sonarqube is running
$ brew services stop sonarqube
$ ps aux | grep sonar # Check that sonarqube is still running
````
**Observed behavior:**
Sonarqube is still runnning after `brew services stop sonarqube`

**Desired behavior:**
All Sonarqube processes should be stopped after `brew services stop sonarqube`

**Comment by VonUniGE** [comment]
A related issue is that status and plist file are not detected by `brew services list`:

```
$ brew services list
Name         Status  User File
sonarqube     vonunige
```

**Comment by MikeMcQuaid** [comment]
If this only affects `sonarqube`: this should be a homebrew-core issue instead. Let me know and I can move it.

**Comment by VonUniGE** [comment]
Yes, this is related to the plist file of Sonarqube. When Sonarqube is running correctly, we have:
```
$ launchctl list | grep sonar
-	0	homebrew.mxcl.sonarqube
```
The first column should show the pid.

**Comment by SMillerDev** [comment]
Yeah, it seems like this never worked correctly. The run command needs to use `sonar console` because `sonar start` will exit immediately.
